src/OVAL/CMakeLists.txt

```diff
@@ -0,0 +1,105 @@
+add_subdirectory("results")
+add_subdirectory("adt")
+add_subdirectory("probes/SEAP")
+file(GLOB_RECURSE PUBLIC_HEADERS "public/*.h")
+
+set (OVAL_SOURCES
+    "collectVarRefs.c"
+    "collectVarRefs_impl.h"
+    "oval_agent.c"
+    "oval_session.c"
+    "oval_defModel.c"
+    "oval_sysModel.c"
+    "oval_affected.c"
+    "oval_agent_api_impl.h"
+    "oval_behavior.c"
+    "oval_component.c"
+    "oval_criteriaNode.c"
+    "oval_definition.c"
+    "oval_definitions_impl.h"
+    "oval_entity.c"
+    "oval_enumerations.c"
+    "oval_filter.c"
+    "oval_generator.c"
+    "oval_glob_to_regex.c"
+    "oval_glob_to_regex.h"
+    "oval_message.c"
+    "oval_object.c"
+    "oval_objectContent.c"
+    "oval_parser.c"
+    "oval_parser_impl.h"
+    "oval_recordField.c"
+    "oval_reference.c"
+    "oval_directives.c"
+    "oval_directives_impl.h"
+    "oval_schema_version.c"
+    "oval_set.c"
+    "oval_state.c"
+    "oval_stateContent.c"
+    "oval_sysEnt.c"
+    "oval_sysInfo.c"
+    "oval_sysInterface.c"
+    "oval_sysItem.c"
+    "oval_syschar.c"
+    "oval_syscharIterator.c"
+    "oval_system_characteristics_impl.h"
+    "oval_test.c"
+    "oval_value.c"
+    "oval_variable.c"
+    "oval_variableBinding.c"
+    "oval_sys_parser.c"
+    "oval_varModel.c"
+    "oval_vardefMapping.c"
+    "oval_version.c"
+)
```

Comment:
why not glob this?
is it intentional that there are private headers and sources in the same list?